apps/sms/views/shared/test/unit/settings_test.js

```diff
@@ -159,7 +167,12 @@ suite('Settings >', function() {
         'operatorResource.sms.maxConcat',
         10
       );
+      triggerSettingsReqSuccess('messages.mms.sendReadReport.enabled', true);
+
+      Settings.sendReadReport.then((result) => assert.equal(result, true)
+        ).then(done, done);
 
+      sinon.assert.calledWith(Settings.initSendReadReport, true);
```

Comment:
not sure this is useful :) the previous assertion tests the same thing, and in a better way